src/coprocessor/dag/executor/table_scan.rs

```diff
@@ -91,7 +91,13 @@ impl TableScanExecutor {
     }
 
     fn new_scanner(&self, range: KeyRange) -> Result<Scanner> {
-        Scanner::new(&self.store, ScanOn::Table, self.desc, false, range).map_err(Error::from)
+        Scanner::new(
+            &self.store,
+            ScanOn::Table,
+            self.desc,
+            self.col_ids.is_empty(),
```

Comment:
What about handle?

Reply:
`handle` wouldn't appear in `self.col_ids` since it was filtered on L47 @BusyJay